src/eth/SmartContractService.js

```diff
@@ -22,11 +24,100 @@ function wrapContract(contract, name, abi, txManager) {
     {},
     {
       get(target, key) {
-        if (nonConstantFns[key] && txManager) {
-          return (...args) =>
-            txManager.createHybridTx(contract[key](...args), {
-              metadata: { contract: name, method: key, args }
+        // Get just the method name (in case name + args e.g. "method(arg1,arg2)" is used as key to call this func
```

Comment:
@ethanbennett @michaelelliot I imagine a lot of this is going to end up in `TransactionManager.formatHybridTx` after the merge

Reply:
Yeah. I've been working on a new branch that integrates with formatHybridTx and also properly builds and deploys the proxy smart contracts in the deployment script, so I'll close this PR and open a new one with those changes soon.

Reply:
Yeah. I've been working on a new branch that integrates with `formatHybridTx` and also properly builds and deploys the proxy smart contracts in the deployment script, so I'll close this PR and open a new one with those changes soon.